teams/wg-allocators.toml

```diff
@@ -0,0 +1,26 @@
+name = "wg-allocators"
+kind = "working-group"
+subteam-of = "libs"
+
+[people]
+leads = [
+    "TimDiekmann"
+]
+members = [
+    "Amanieu",
+    "CAD97",
+    "Lokathor",
+    "scottjmaddox",
+    "TimDiekmann",
+    "Wodann"
+]
+
+[website]
+page = "allocators"
```

Comment:
@pietroalbini this `page` key is also for top-level teams, right ?

Reply:
That shouldn't be needed here, the `page` key exists as a workaround for other teams.